docs/whats-new.md

```diff
@@ -12,12 +12,12 @@ Release date: March 31, 2017
 - **New Examples** deck.gl now provides multiple stand-alone examples, with minimal
   configuration files (`package.json`, `webpack.config.js` etc) intended to make it
   easy to just copy an example folder and get an app up and running in minutes.
-- **Unified 64-bit Layers** - 64-bit Layers are now unified with 32-bit layers, use `fp64` prop.
+- **Unified 64-bit Layers** - 64-bit Layers are now unified with 32-bit layers, controlled via a `fp64` prop.
```

Comment:
**an** `fp64` prop (assuming f is pronounced as "ef")

Or just say: a new `fp64`